### ISSUE TITLE
Unable to obtain dependencies with renv

### ISSUE THREAD

**Issue by ischmidt20** [rstudio/rsconnect#951]
I am trying to publish a Shiny application to a Posit Connect server. I am using `renv` version 1.0.1 for package management, the lockfile looks good, and renv reports that nothing is out of sync. However, whenever I try to publish, I see the following error:

`Error in if (pkg$Repository == "CRAN") { : argument is of length zero`

I have traced this issue to the call to `rsconnect::appDependencies()`, as I see the same error when calling that function from the console. Here is the full traceback:

```
6: FUN(X[[i]], ...)

5: lapply(packages, standardizeRenvPackage, availablePackages = availablePackages,

       biocPackages = biocPackages, repos = repos)

4: standardizeRenvPackages(renv$Packages, repos, biocPackages = biocPackages(bundleDir))

3: parseRenvDependencies(bundleDir)

2: computePackageDependencies(bundleDir, extraPackages, quiet = TRUE)

1: rsconnect::appDependencies()
```

Any idea what could be causing this, or if there is a workaround? At the moment it is preventing me from publishing my Shiny app.

**Comment by aronatkins** [comment]
The particular line of code you reference was adjusted in https://github.com/rstudio/rsconnect/pull/931

Could you try using the latest development version of rsconnect to see if that helps or produces better feedback?

```r
remotes::install_github("rstudio/rsconnect")
```

Additionally, any chance that you could create a reprex to help us produce the same error? It would be useful to see your `getOption("repos")` and how packages were installed.

**Comment by ischmidt20** [comment]
> Could you try using the latest development version of rsconnect to see if that helps or produces better feedback?

This worked, thanks!

```
> getOption("repos")
                         CRAN 
"https://cloud.r-project.org"
```

Upon installing the dev version, `rsconnect::appDependencies()` now returns the appropriate data frame with Repository equal to `"https://cloud.r-project.org"` for all packages (with the exception of rsconnect).

**Comment by aronatkins** [comment]
The version you tested was released to CRAN as rsconnect-1.0.2. Please let us know if you continue to have problems.
